js/main.js

```diff
@@ -1,7 +1,72 @@
+    liveLink: 'https://ybadr99.github.io/portfolio/',
+    sourceLink: 'https://github.com/ybadr99/portfolio',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, ",
+    featuredImage: './images/projects/project02.png',
+    technologies: ['Ruby on Rails', 'css', 'JavaScript', 'HTML'],
+    liveLink: 'https://ybadr99.github.io/portfolio/',
+    sourceLink: 'https://github.com/ybadr99/portfolio',
+  },
+  {
+    id: 3,
+    name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, ",
+    featuredImage: './images/projects/project03.png',
+    technologies: ['Ruby on Rails', 'css', 'JavaScript', 'HTML'],
+    liveLink: 'https://ybadr99.github.io/portfolio/',
+    sourceLink: 'https://github.com/ybadr99/portfolio',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, ",
+    featuredImage: './images/projects/project02.png',
+    technologies: ['Ruby on Rails', 'css', 'JavaScript', 'HTML'],
+    liveLink: 'https://ybadr99.github.io/portfolio/',
+    sourceLink: 'https://github.com/ybadr99/portfolio',
+  },
+  {
+    id: 5,
+    name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, ",
+    featuredImage: './images/projects/project03.png',
+    technologies: ['Ruby on Rails', 'css', 'JavaScript', 'HTML'],
+    liveLink: 'https://ybadr99.github.io/portfolio/',
+    sourceLink: 'https://github.com/ybadr99/portfolio',
+  },
+  {
+    id: 6,
+    name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, ",
+    featuredImage: './images/projects/project02.png',
+    technologies: ['Ruby on Rails', 'css', 'JavaScript', 'HTML'],
+    liveLink: 'https://ybadr99.github.io/portfolio/',
+    sourceLink: 'https://github.com/ybadr99/portfolio',
+  },
+
+];
+
 const menuIcon = document.querySelector('.menu-icon');
 const mobileMenu = document.querySelector('.mobile-nav-items');
 const navItem = document.querySelectorAll('.nav-item');
 
+const projectsContainer = document.getElementById('projects');
+// popup window
+const modal = document.querySelector('#popup-window');
+const close = document.querySelector('.close');
+const modalContent = document.querySelector('.modal-content');
```

Comment:
- [ ] Very nice implementation :clap: but if you see your design is not the same as Figma design please stick to the Figma design check below :point_down:  the screenshot of Figma design and video of your design :slightly_smiling_face:  

| Desktop   |      Mobile      |
|---------- |:----------------:|
| ![Screenshot from 2023-04-13 12-01-08](https://user-images.githubusercontent.com/45226848/231679745-c813835b-e41f-4bd2-a2e4-4aaee1a21d1c.png)  |  ![Screenshot from 2023-04-13 12-02-00](https://user-images.githubusercontent.com/45226848/231679855-319cce26-fac0-4fb7-a3f6-b6668d2be8b4.png) | 

[screen-capture (5).webm](https://user-images.githubusercontent.com/45226848/231679989-5e97582f-e764-44d2-98ba-6568f2425927.webm)

- [ ] If you check your design it's totally different from the Figma design so kindly fix it :+1:

Reply:
> * [ ]  Very nice implementation 👏 but if you see your design is not the same as Figma design please stick to the Figma design check below 👇  the screenshot of Figma design and video of your design 🙂
> 
> Desktop	Mobile
> ![Screenshot from 2023-04-13 12-01-08](https://user-images.githubusercontent.com/45226848/231679745-c813835b-e41f-4bd2-a2e4-4aaee1a21d1c.png)	![Screenshot from 2023-04-13 12-02-00](https://user-images.githubusercontent.com/45226848/231679855-319cce26-fac0-4fb7-a3f6-b6668d2be8b4.png)
> * [ ]  If you check your design it's totally different from the Figma design so kindly fix it 👍

- [ ] As the previous reviewer said above, your design is overflowing in it's container, kindly adjust the portfolios' image sizes to create space for the other items inside the container. 😉